src/olympia/addons/models.py

```diff
@@ -664,18 +664,20 @@ def valid_file_statuses(self):
             return [amo.STATUS_PUBLIC]
         return amo.VALID_FILE_STATUSES
 
-    def find_latest_public_version(self):
-        """Retrieve the latest public version of an addon.
+    def find_latest_public_listed_version(self):
```

Comment:
My preference would be not changing as it's already pretty wordy but I didn't consider the status name ambiguity so up to you.